### PR TITLE
cni cilium: add compatibililty matix for ipvs proxy mode

### PR DESCRIPTION
**Pull request by vgramer**
add compatibililty matix for cilium with  ipvs proxy mode

### CODE REVIEW

File: content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md [hunk 1]
```diff
@@ -78,8 +78,23 @@ The following table lists the supported operating systems and cloud providers fo
...
+| AWS                   | ~[^1]  | x      | ~[^1]   | ~[^1]       |
+| Azure                 | ~[^1]  | x      | ~[^1]   | ~[^1], [^2] |
+| Google Cloud Platform | ~[^1]  | -      | -       | -           |
+| Openstack             | ~[^1]  | x      | ~[^1]   | ~[^1]       |
+**NOTE:**
+
+- A hyphen(-) denotes that the operating system is not supported for the given cloud provider.
+- A tilde(~) denotes a partial support
+
+[^1]: Pod can not access to <internal_node_ip>:<node_port>. issue [8767](https://github.com/kubermatic/kubermatic/issues/8767)
```
Comment (SimonTheLeg) [inline content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md:95]:
```suggestion
[^1]: Pod can not access <internal_node_ip>:<node_port>. issue [8767](https://github.com/kubermatic/kubermatic/issues/8767)
```

File: content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md [hunk 2]
```diff
@@ -78,8 +78,23 @@ The following table lists the supported operating systems and cloud providers fo
...
+[^2]: LoadBalancer service are not accessible from outside the cluster. issue [8768](https://github.com/kubermatic/kubermatic/issues/8768)
```
Comment (SimonTheLeg) [inline content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md:96]: either
```suggestion
[^2]: LoadBalancer services are not accessible from outside the cluster. issue [8768](https://github.com/kubermatic/kubermatic/issues/8768)
```
or
```suggestion
[^2]: LoadBalancer service is not accessible from outside the cluster. issue [8768](h

**Review by SimonTheLeg — CHANGES_REQUESTED** [review]

File: content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md (hunk 2 above)
Comment (rastislavs) [inline content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md:96]: To my understanding, #8768 is more a generic KKP Azure + RHEL issue, not specific to Cilium. Could you check if it is present on Canal as well? In that case, I would not list it here in CIlium-specific docs..

Comment (vgramer) [reply content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md:96]: sure, i am testing that

File: content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md (hunk 1 above)
Comment (vgramer) [reply content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md]: done :)

File: content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md (hunk 2 above)
Comment (vgramer) [reply content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md]: as expected, this issue is not related to cilium but  Azure + RHEL issue setup. I have updated issue kubermatic/kubermatic#8768

Comment (rastislavs) [reply content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md]: thanks!

Comment (vgramer) [reply content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md]: removed all bullet according to @rastislavs  [comment](https://github.com/kubermatic/docs/pull/922#discussion_r790505940)

**Comment by rastislavs** [comment]
Can I ask one more favor? Could we move these compatibility tables up in the CIlium docs section, right under the first paragraph mentioning:

> but requires more recent kernel versions on the worker nodes (see Cilium System Requirements).

To make it more visible, as it is quite important.

**Pull request merged**